docs/src/main/asciidoc/security-architecture.adoc

```diff
@@ -49,7 +49,7 @@ In other contexts, it is possible to have other parallel representations of the
 For more information, see the Quarkus xref:security-identity-providers.adoc[Identity providers] guide.
 
 === `SecurityIdentityAugmentor`
-Because Quarkus Security is customizable, for example, you can add authorization roles to `SecurityIdentity`, you can register and prioritize one or more custom security augmentors.
+Because Quarkus Security is customizable, for example, you can add authorization roles to `SecurityIdentity` and register and prioritize one or more custom security augmenters.
```

Comment:
This is an interesting one, interface is called `SecurityAugmentor`. May be:

one or more custom `SecurityAugmentor` implementations ?

Reply:
Updated. Thanks.